zenodo/config.py

```diff
@@ -172,6 +172,8 @@ def _(x):
 ACCOUNTS_SESSION_REDIS_URL = "redis://localhost:6379/2"
 #: Cache for storing access restrictions
 ACCESS_CACHE = 'zenodo.modules.cache:current_cache'
+#: Disable JSON Web Tokens
+ACCOUNTS_JWT_ENABLE=False
```

Comment:
We need to test it properly first, since I'm not sure we have deposit-js released yet.